Index: local.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>## This file must *NOT* be checked into Version Control Systems,\n# as it contains information specific to your local configuration.\n#\n# Location of the SDK. This is only used by Gradle.\n# For customization when using a Version Control System, please read the\n# header note.\n#Thu Apr 03 19:18:59 GMT-03:00 2025\nsdk.dir=C\\:\\\\Users\\\\Hanny\\\\AppData\\\\Local\\\\Android\\\\Sdk\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/local.properties b/local.properties
--- a/local.properties	(revision cb3a1bd5fac298b67a305af146274efb9b31c583)
+++ b/local.properties	(date 1743963380899)
@@ -4,5 +4,5 @@
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
-#Thu Apr 03 19:18:59 GMT-03:00 2025
-sdk.dir=C\:\\Users\\Hanny\\AppData\\Local\\Android\\Sdk
+#Sun Apr 06 15:16:20 BRT 2025
+sdk.dir=/Users/arthurmartinstorres/Library/Android/sdk
